spark-connector/spark-common/src/test/java/org/apache/gravitino/spark/connector/integration/test/sql/TestCase.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.spark.connector.integration.test.sql;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import lombok.Getter;
+import lombok.ToString;
+
+/** A test SQL files which contains multi SQL queries. */
+@Getter
+@ToString
+public class TestCase {
+  private Path testFile;
```

Comment:
You'd better make some variables `final` here and below - if they will not modified during running.

Reply:
done